x-pack/docs/en/security/securing-communications/index.asciidoc

```diff
@@ -8,6 +8,8 @@ manipulation of the data, and attempts to gain access to the server and thus the
 files storing the data. Securing your nodes helps reduce the risk from
 network-based attacks.
 
+WARNING: A full description of TLS/SSL, PKI (Public Key Infrastructure) certificates, and Certificate Authority is beyond the scope of this document. This page assumes prior knowledge of TLS/SSL as well as access to valid certificates.
+
```

Comment:
This final part (`access to valid certificates`) doesn't seem right. The reader might want to use their own certificates, but we don't assume that they have them, as we provide options to create them.